common-pipeline/operators/uihints-v3-schema.json

```diff
@@ -283,13 +283,24 @@
             "datepickerRange",
             "timefield",
             "timestampfield",
+            "slider",
             "code",
             "list",
             "hidden"
           ]
         },
         "increment": {
-          "description": "Determines the increment/decrement value for the spinner control only",
+          "description": "Determines the step value to increment/decrement for the slider control",
```

Comment:
This should be spinner and slider control.

Reply:
Sure. I have updated accordingly.